sources/Application/Instruments/MidiInstrument.cpp

```diff
@@ -148,6 +149,11 @@ void MidiInstrument::ProcessCommand(int channel, FourCC cc, ushort value) {
     }
   } break;
 
+  case I_CMD_VELM: {
+    // VELM cmds set velocity for MIDI steps
+    velocity_ = int((value + 0.99) / 2);
```

Comment:
nit: I feel `floor(value/2)` would show intent better than this and look cleaner

Reply:
I did wonder if I should change it as that's what the original lgpt code used, I'll fix it up 👍